### ISSUE TITLE
Replicate Original version's mAP

### ISSUE THREAD

**Issue by kenrubiooo** [YunYang1994/tensorflow-yolov3#78]
Hi, @YunYang1994 

What parameters should I use to get a similar mAP of the original implementation? What iou_thresh and score_thresh values should I use? The implementation from AlexeyAB has parameters ignore_thresh and truth_thresh, are these 4 related to one another? Tou!hank y

**Comment by hitzht** [comment]
> Hi, @YunYang1994
> 
> What parameters should I use to get a similar mAP of the original implementation? What iou_thresh and score_thresh values should I use? The implementation from AlexeyAB has parameters ignore_thresh and truth_thresh, are these 4 related to one another? Tou!hank y

What's your mAP in your implementation based on VOC datasets? I just get 33% with data augmentation.

**Comment by CR-Ko** [comment]
@hitzht how about your mAP of VOC without augmentation?
And in what condition? steps? LR ?

I am still training on VOC but didn't get mAP > 0.2 on test set....

**Comment by hitzht** [comment]
1080Ti batch32 step40000 lr0.001,mAP=0.33

**Comment by CR-Ko** [comment]
@hitzht 
Do u mean using evaluate.py on test set?

**Comment by hitzht** [comment]
> @hitzht
> Do u mean using evaluate.py on test set?

yeah
